coordinape/coordinape
# new GIVE page

## Motivation and Context

We designed a new allocation/give workflow as part of 1.5 design work. This is it!

## Description

New GIVE page w/ table view, autosave, lots of room for future improvement. 

## Test and Deployment Plan

Run locally and try to go through allocation using the new UI. It's linked from the old allocate page in the allocation stepper if there is a current epoch. You should also try: 

* check out the page with no current epoch
* try various combinations of: 
** team selection disabled shouldn't matter, anyone can manage collaborators
** blocked from giving
** blocked from receiving
** opted out of giving
** no contributions, yes contributions
** no epoch statement, yes epoch statement
** 0 starting tokens, more than 100 starting tokens
* try weird timing stuff on the autosave
* toggling collaborators, paging through users

## Screenshots (if appropriate):

<img width="1199" alt="image" src="https://user-images.githubusercontent.com/95149165/192613981-d38cf420-0220-45c7-9119-5b8972c540a4.png">

<img width="1202" alt="image" src="https://user-images.githubusercontent.com/95149165/192614084-e40e8132-6558-4929-9209-98d677c9c763.png">


## Reviewers

@ivyquinzel @wingfeatherwave @topocount @zashton @levity @reeserj 

## Related Issue
Fixes #1162 
Fixes #928 
Fixes #926 
Fixes #927
Fixes #925 
Fixes #830 

## Out of scope and handling after:

More filtering and sorting
#929 
#1400 
#1401 
#1402
